### ISSUE TITLE
Some containers could stop faster

### ISSUE THREAD

**Issue by heyyo-droid** [wodby/docker4drupal#552]
I noticed some containers are stopping after around 10.5 seconds like cron and adminer(maybe other).
10 seconds is the maximal time docker is waiting before stopping the container if unresponsive.  

It looks like those containers don't receive SIGTERMs.

Adding `init: true` fixed this for me:  
https://docs.docker.com/compose/compose-file/compose-file-v2/#init

What do you think of adding it ?

**Comment by csandanov** [comment]
yes, we can add it for crond and adminer, it looks like crod doesn't handle it well when running in foreground
adminer is running on a built-in php server which I guess is very limited

**Comment by csandanov** [comment]
Adding it to adminer helped but crond show the following error 
```
[FATAL tini (1)] Unexpected error when forwarding signal: 'Operation not permitted'
```
but still works – container shuts down instantly

let's keep it like that for now

**Comment by heyyo-droid** [comment]
Right I didn't pay attention to this error.
Thanks
